drivers/cas_ce_events/README.md

```diff
@@ -1,3 +1,16 @@
-## CasCeEvents README
+# CasCeEvents
 
-This README file should be used to explain the functionality of the driver.
+Propagate CAS generated CE events from CAS generated  referral events
+(in the `cas_referral_events` table) into the synthetic events table, and
+finally as sythetic events in the HUD events table.
+
+## CAS Configuration
+
+In order to attach synthetic events to enrollments, CAS Programs must be
+associated with HMIS Projects.
+
+## Warehouse Configuration
+
+To distinguish synthetic events, they are stored in a warehouse data source
```

Comment:
I'm tempted to have a first_or_create on the CAS data source so we sync if there's anything to sync and don't fail for lack of configuration...